packages/lambda-powertools-cloudwatchevents-client/package.json

```diff
@@ -21,4 +22,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
+}
```

Comment:
Nitpicky I know but could we get the newlines re-added? (a few repeated examples in this PR)
```suggestion
}

```

Reply:
All done!